test/alternator/test_table.py

```diff
@@ -553,3 +554,59 @@ def creates():
                     time.sleep(1)
                     continue
                 raise
+
+# Test that DeleteTable returns correct TableDescription (issue #11472)
+def test_delete_table_description(dynamodb):
+    table_name = unique_table_name()
+
+    table = create_table(dynamodb, table_name)
+    table.meta.client.get_waiter('table_exists').wait(TableName=table_name)
+    got = table.delete()['TableDescription']
+    table.meta.client.get_waiter('table_not_exists').wait(TableName=table_name)
```

Comment:
nitpick: you did not have to wait for the tale_not_exists event. You care about the response from table.delete() (which you saved in "got") - you don't care when the actual deletion finishes.

Reply:
Done